code/index.html

```diff
@@ -3,28 +3,116 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Business Site</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+<!-- HEADER-->
+    <header class="header"> 
+      <video playsinline autoplay muted loop >
+        <source src="./videos/video.mp4" type="video/mp4">
+      </video>
+      <div class="overlay">
+        <h1>
+          Book your dream vacation here
+        </h1>
+        <h2>Only one click away</h2>
+        <button>BOOK NOW</button>
+      </div>
+    </header>
+
+
+<!--MAIN SECTION-->
+  <main class="container">
+    <section>
+    <!--FORM-->
+      <form action="http://httpbin.org/anything" method="POST">
+          <fieldset>
+              <legend>Book now</legend>
+  
+              <label for="fistName">First Name</label>
+              <input type="text">
+
+              <label for="lastName">Last Name</label>
+              <input type="text">
+
+              <label for="email">Email</label>
+              <input type="email">
+
+              <label for="date">Date</label>
+              <input type="date">
+
+              <label>Choose your destination:
+                <select class="options" name="country">
+                    <option value="1">Creta</option>
+                    <option value="2">Malaga</option>
+                    <option value="3">Rhodos</option>
+                    <option value="4">Gran Canaria</option>
+                </select>
+              </label>
+
+              <div>
+                  <p class="labels-radio">Choose one booking option:</p>
+                  <input type="radio" name="radio">
+                  <label for="">Hotel</label>
+
+                  <input type="radio"name="radio">
+                  <label for="">Hotel/Flight</label>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+                  <input type="radio" name="radio">
+                  <label for="">All inclusive</label>
+              </div>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
+              <div class="checks">
+                <input type="checkbox">
```

Comment:
This could use the `required` tag: `<input type="checkbox" required>` to make sure users who submit the form also agree to the terms.

Reply:
Yes! Forgot about that one :)